content/erguotou/drone-instant-access.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 2020-03-07T00:00:00+08:00
+title: InstantAccess(即时达)
+author: erguotou
```

Comment:
The author is so far matching the GitHub handle, so this should be `erguotou520 `

Reply:
fixed